src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
In this case, you have duplicate element `<div class="stars__star"></div>`. For cases with duplication, it is better to use a list, not div.